b2b/tasks.py

```diff
@@ -12,3 +12,11 @@ def queue_enrollment_code_check(contract_id: int):
 
     contract = ContractPage.objects.get(id=contract_id)
     ensure_enrollment_codes_exist(contract)
+
+
+@app.task()
```

Comment:
Should this be `acks_late=True` because if the worker gets killed in the middle of a task it won't rerun for possibly another 24 hours?